### PR TITLE
feature: ZENKO-3863 Upgrade to Node 16

### PR DESCRIPTION
**Pull request by miniscruff**

### CODE REVIEW

**Review by tcarmet — APPROVED** [review]

**Review by rachedbenmustapha — APPROVED** [review]

File: test/ZenkoClient.test.js [hunk 1]
```diff
@@ -395,38 +397,41 @@ describe('class ZenkoClient JSON api', () => {
...
-                    return done();
-                }).catch(err => {
-                    expect(err).not.toBeTruthy();
-                    return done();
-                });
-        });
+            done => {
+                zenkoClient.getLocationsStatus().promise()
+                    .then(res => {
+                        expect(res).toEqual(
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]:
```suggestion
                        expect(res).toEqual({ response: 'get crr test response' });
```

File: test/ZenkoClient.test.js [hunk 2]
```diff
@@ -395,38 +397,41 @@ describe('class ZenkoClient JSON api', () => {
...
-                    return done();
-                }).catch(err => {
-                    expect(err).not.toBeTruthy();
-                    return done();
-                });
-        });
+            done => {
+                zenkoClient.getLocationsIngestionStatus().promise()
+                    .then(res => {
+                        expect(res).toEqual(
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: same here.

File: test/ZenkoClient.test.js [hunk 3]
```diff
@@ -469,8 +475,8 @@ describe('class ZenkoClient JSON api', () => {
             }).promise()
                 .then(res => {
                     expect(res).toEqual({
-                        response: `ingestion pause ${expectedSite} ` +
-                                  'test response',
+                        response: `ingestion pause ${expectedSite} `
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: Same here, single line.

File: test/ZenkoClient.test.js [hunk 4]
```diff
@@ -566,8 +572,8 @@ describe('class ZenkoClient JSON api', () => {
             }).promise()
                 .then(res => {
                     expect(res).toEqual({
-                        response: 'ingestion scheduled resume for ' +
-                                  `${expectedSite} test response`,
+                        response: 'ingestion scheduled resume for '
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: Same.

File: test/ZenkoClient.test.js [hunk 5]
```diff
@@ -549,8 +555,8 @@ describe('class ZenkoClient JSON api', () => {
             }).promise()
                 .then(res => {
                     expect(res).toEqual({
-                        response: 'crr scheduled resume for ' +
-                                  `${expectedSite} test response`,
+                        response: 'crr scheduled resume for '
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: Same.

File: test/ZenkoClient.test.js [hunk 6]
```diff
@@ -530,8 +536,8 @@ describe('class ZenkoClient JSON api', () => {
             }).promise()
                 .then(res => {
                     expect(res).toEqual({
-                        response: `ingestion resume ${expectedSite} ` +
-                                  'test response',
+                        response: `ingestion resume ${expectedSite} `
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: Same.

File: test/ZenkoClient.test.js [hunk 7]
```diff
@@ -395,38 +397,41 @@ describe('class ZenkoClient JSON api', () => {
...
+            });
     });
 
     describe('ZenkoClient::pauseAllSites', () => {
         it('should return correct response for crr service', done => {
             zenkoClient.pauseAllSites().promise()
                 .then(res => {
                     expect(res).toEqual(
-                        { response: 'crr pause test response' });
+                        { response: 'crr pause test response' },
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: Here too.

File: test/ZenkoClient.test.js [hunk 8]
```diff
@@ -438,7 +443,8 @@ describe('class ZenkoClient JSON api', () => {
             zenkoClient.pauseAllIngestionSites().promise()
                 .then(res => {
                     expect(res).toEqual(
-                        { response: 'ingestion pause test response' });
+                        { response: 'ingestion pause test response' },
```
Comment (naren-rajendran) [inline test/ZenkoClient.test.js]: Same.

**Review by naren-rajendran — COMMENTED** [review]
Looks good to me, except for few minor cosmetics.

**Review by naren-rajendran — APPROVED** [review]

**Pull request merged**